Misc/NEWS.d/next/Library/2024-08-18-08-25-32.gh-issue-123084.rf8izX.rst

```diff
@@ -0,0 +1,2 @@
+:class:`!shutil.ExecError` is now a deprecated alias of
+:class:`RuntimeError`.
```

Comment:
Though this exception isn't documented, it's presented in `__all__` so I think it's ok to have a `NEWS` entry here.
Additionally, I think we should add a note about this deprecation to `Doc/deprecations/pending-removal-in-future.rst` or `Doc/deprecations/pending-removal-in-3.16.rst`, this depends on the fact when this exception will be removed.

Reply:
Let's hear Barney's opinion on that. Personally I think it's ok to remove this exception in 3.16.

Reply:
I think we should explicitly slate it for removal in 3.16, so that the module `__getattr__()` doesn't stick around for too long.

Reply:
Would you like me to move it `Doc/deprecations/pending-removal-in-3.16.rst`, or is the "future" one fine?

Reply:
> Would you like me to move it `Doc/deprecations/pending-removal-in-3.16.rst`

Yes please